torch/distributed/__init__.py

```diff
@@ -54,6 +56,32 @@ def is_available() -> bool:
         _make_nccl_premul_sum,
     )
 
+    def breakpoint(rank: int = 0):
```

Comment:
What happens if every rank calls this API with their own local `rank` argument? All of them would call `breakpoint` and then would pdb just return you the debugger for just one of them?